lib/mutant/ast/meta/send.rb

```diff
@@ -29,6 +29,13 @@ def proc?
           naked_proc? || proc_new?
         end
 
+        # Test if message receiver is :T
+        #
+        # @return [Boolean]
+        def sorbet_receiver?
+          receiver.to_s == '(const nil :T)'
```

Comment:
Is there a better way for doing this? Also, should I add `::T` too?

Reply:
You might be able to do `receiver.eql?(s(:const, nil, :T))`. If you can't, you may need to mix in `Mutant::AST::Sexp` (or, i suppose, you could also manually initialize `Parser::AST::Node` but it's probably better to mix in).

I think you need to check the `::T` case as well if you want that to be recognized and not mutated, but mutant will also strip off unnecessary `::T` prefixes so that would only matter in fairly rare cases where you shadow `T`.

Reply:
I'd assign it to a constant:

```
SORBET_RECEIVER = s(:const, nil, :t)
```

no need to re-construct the ast node each time.